CODE_OF_CONDUCT.md

```diff
@@ -1,2 +1,84 @@
-# Insert code of conduct here
-# When in doubt, checkout https://www.contributor-covenant.org/
+# Code of Conduct
+
+## 1. Purpose
+
+A primary goal of Developers Italia is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).
```

Comment:
We might want to drop the mentions to "Developers Italia" and use something else. "this project"?

Reply:
yes, fine for me to say "this project"

Reply:
Resolved with new commit.